lib/msal-angular/docs/FAQ.md

```diff
@@ -110,7 +110,7 @@ For those using the `HashLocationStrategy`, we recommend:
 - Making sure the `MsalRedirectComponent` is bootstrapped
 - Optionally: adding `MsalGuard` to all your routes if you want all your routes protected
 
-See our [Angular 11 sample](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/samples/msal-angular-v2-samples/angular11-sample-app/src/app/app-routing.module.ts) for examples of how to protect routes with `MsalGuard`.
+See our [Angular 11 sample](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/msal-lts/samples/msal-angular-v2-samples/angular11-sample-app/src/app/app-routing.module.ts) for examples of how to protect routes with `MsalGuard`.
```

Comment:
Are we going to replace these with newer samples for v3?

Reply:
I have replaced as many with newer samples as possible. As we get more samples for v3, I may be able to phase out more older samples in docs.